tests/test_reforrmat_interaction_network_graph.py

```diff
@@ -0,0 +1,128 @@
+import MDAnalysis
+import networkx
+import numpy as np
+import os
+import pytest
+from numpy.testing import assert_array_equal
+from pyinteraph.reformat_interaction_network_graph import ReformatDatGraph
+
+
+@pytest.fixture
+def change_test_dir(request):
+    os.chdir('tests/data/single_chain')
+    yield
+    os.chdir(request.config.rootdir)
+
+
+@pytest.fixture
+def reference_structure():
+    return 'sim.prot.A.pdb'
+
+
+@pytest.fixture
+def interaction_network_file():
+    return 'hb-graph-filtered.dat'
+
+
+@pytest.fixture
+def empty_interaction_network_file():
+    return 'xx.dat'
```

Comment:
can we give it a more explicit name than `xx.dat`? Also it needs to be added as a commit